File: vdp-php/tests/pymntacntattrinqry/GeneralAttributesInquiryTest.php
<?php

namespace Vdp;

class GeneralAttributesInquiryTest extends \PHPUnit_Framework_TestCase {
	
	public function setUp() {
		$this->visaAPIClient = new VisaAPIClient;
		$this->generalAttributeInquiry = json_encode ( [
		  "primaryAccountNumber" => "4465390000029077"
	]);
	}
	public function testGeneralAttributesEnquiry() {
		$baseUrl = "paai/";
		$resourcePath = "generalattinq/v1/cardattributes/generalinquiry";
		$statusCode = $this->visaAPIClient->doMutualAuthCall ( 'post', $baseUrl.$resourcePath, 'General Attributes Inquiry call', $this->generalAttributeInquiry );
		$this->assertEquals($statusCode, "200");
	}
}

File: vdp-c-sharp/vdp-c-sharp/VDPApiTest/pymntacntattrinqry/GeneralAttributesInquiryTest.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace Vdp
{
    [TestClass]
    public class GeneralAttributesInquiryTest
    {
        private string generalAttributeInquiry;
        private VisaAPIClient visaAPIClient;

        public GeneralAttributesInquiryTest()
        {
            visaAPIClient = new VisaAPIClient();
            generalAttributeInquiry =
                    "{"
                      + "\"primaryAccountNumber\": \"4856200001123821\""
                  + "}";
        }

        [TestMethod]
        public void TestGeneralAttributesEnquiry()
        {
            string baseUri = "paai/";
            string resourcePath = "generalattinq/v1/cardattributes/generalinquiry";
            string status = visaAPIClient.DoMutualAuthCall(baseUri + resourcePath, "POST", "General Attributes Inquiry Test", generalAttributeInquiry);
            Assert.AreEqual(status, "OK");
        }
    }
}
